README.md

```diff
@@ -34,15 +37,17 @@ https://github.com/cschiller/zhongwen
 
 ## Installation
 
-The Zhongwen Chrome extension is available via the Chrome Web Store:
+The Zhongwen extension is available as Chrome and Firefox Add-on:
```

Comment:
Wording:

Zhongwen is available as a Chrome extension as well as a Firefox add-on.